app/logic/Meet/LiveKit/LiveKitMediaDeviceStreams.ts

```diff
@@ -29,6 +29,7 @@ export class LiveKitMediaDeviceStreams extends MediaDeviceStreams {
   }
 
   async setCameraOn(on: boolean, device?: string) {
+    const { Track } = await import("livekit-client");
```

Comment:
You need `Track` only for the `Track.Source` enum, right?

Can you import only that enum, not the rest of the module?
If not, can you copy that enum into the class as `TrackSource`?

That would avoid most changes here.